dd-sdk-android-internal/src/test/java/com/datadog/internal/utils/StringBuilderExtKtTest.kt

```diff
@@ -0,0 +1,74 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.internal.utils
+
+import com.datadog.android.internal.utils.appendIfNotEmpty
+import fr.xgouchet.elmyr.annotation.StringForgery
+import fr.xgouchet.elmyr.junit5.ForgeExtension
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.junit.jupiter.api.extension.Extensions
+
+@Extensions(
+    ExtendWith(ForgeExtension::class)
+)
+internal class StringBuilderExtKtTest {
+
+    @Test
+    fun `M add char W addIfNotEmpty {buffer is not empty}`(
```

Comment:
nit: the method name is actually a bit different (same for other tests in this file)

```suggestion
    fun `M add char W appendIfNotEmpty {buffer is not empty}`(
```